Resources/Prototypes/_HL/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```
@@ -19,3 +19,45 @@
     damage:
       types:
         Blunt: 8
+
+- type: entity
+  name: SESWC Vortex-12
```

```suggestion
  name: Vortex-12
```
Guns don't typically have their manufacturer name in them